File: Wrap-Up/python/strong_number.py
number = 145
original_number = number
sum_of_factorials = 0

        
while (number > 0):
    digit = number % 10

           
    factorial = 1
    for count in range(1, digit+1):
        factorial = factorial * count
            

        
    sum_of_factorials  = sum_of_factorials  + factorial

    number = number // 10 
        


if (sum_of_factorials  == original_number ):
    print(f"{original_number} is a Strong number.")
       
else:
    print(f"{original_number} is not a Strong number.")
        

File: Wrap-Up/javaScript/strongNUmber.js
let number = 145;
let originalNumber = number;
let sumOfFactorials = 0;

        
while (number > 0) {
    let digit = number % 10; 

           
    let factorial = 1;
    for (let count = 1; count <= digit; count++) {
        factorial = factorial * count;
            }

        
    sumOfFactorials = sumOfFactorials + factorial;

    number = number / 10; 
        }


if (sumOfFactorials == originalNumber) {
    console.log(originalNumber + " is a Strong number.");
        } 
else {
    console.log(originalNumber + " is not a Strong number.");
        }

